### PR TITLE
zcash_client_backend: Generate Orchard component for USKs

### PR DESCRIPTION
**Pull request by str4d**
This ensures that the resulting UFVKs and UAs also contain Orchard components.

### CODE REVIEW

**Review by nuttycom — APPROVED** [review]
utACK

File: zcash_client_backend/src/keys.rs [hunk 1]
```diff
@@ -283,6 +294,11 @@ impl UnifiedFullViewingKey {
         self.sapling.as_ref()
     }
 
+    /// Returns the Orchard full viewing key component of this unified key.
```
Comment (daira) [inline zcash_client_backend/src/keys.rs:297]:
```suggestion
    /// Returns the Orchard full viewing key component of this unified key.
    /// (Orchard full viewing keys are always diversifiable.)
```

Comment (nuttycom) [reply zcash_client_backend/src/keys.rs:297]: I don't understand what it means for a full viewing key to be diversifiable?

Comment (str4d) [reply zcash_client_backend/src/keys.rs:297]: I think @daira was contrasting this to the docstring for the Sapling side, which says it returns a "diversifiable full viewing key". I think a better resolution would be to remove "diversifiable" from the Sapling text, as it's more of an implementation detail that users shouldn't need to care about.

**Pull request merged**